integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartBuildIT.java

```diff
@@ -36,7 +36,8 @@
 
     private static final Path testDirPath = Paths.get("target/quarkus-codestart-build-test");
 
-    private static final Set<String> EXCLUDED = Sets.newHashSet("spring-web-codestart", "picocli-codestart");
+    private static final Set<String> EXCLUDED = Sets.newHashSet("spring-web-codestart", "picocli-codestart",
+            "hibernate-orm");
```

Comment:
```suggestion
            "hibernate-orm-codestart");
```

I wasn't filtering it out because the filtering is using the name (not the ref)